src/components/SignupPage/SignupPage.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+
+class SignupPage extends Component {
```

Comment:
Missing the color part and the checking of a strong and weak password, also the valid email.